sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts

```diff
@@ -27,10 +28,112 @@ enum GapiScopes {
 const initialScopeString = 'profile email';
 
 /**
- * This file contains a collection of functions that interact with gapi.
+ * This module contains a collection of functions that interact with gapi.
  */
 class GapiManager {
 
+  public static drive = class {
```

Comment:
We'd need to use ES6 export modules for that, which I can't get to work with Typescript and Polymer.
Maybe as these grow a little more this would force us to find a solution for that.